research-software-directory/RSD-as-a-service
# Authentication initial setup

# Authentication initial setup

Changes proposed in this pull request:

* Simple setup to create account and login with SURFconext

How to test:

* After building and starting, go to http://localhost:7000/login/surfconext, follow the link and login. You should see a token after returning and in the database there should be an entry for the tables account and login_for_account. The login_for_account entry should have a foreign key to the account that you see if you decode the token you saw before. The token is signed with the hard coded String JWT_SECRET found in authentication/src/main/java/nl/esciencecenter/rsd/authentication/Main.java

PR Checklist:
